src/components/header.js

```diff
@@ -0,0 +1,43 @@
+import React from "react"
```

Comment:
This is not explained in the tutorial or wherever you based toure code of, but remember these 2 rules:

All files containing JSX, should be .jsx files instead of .js
All files exporting a react component should be capitalised